libkbfs/folder_branch_ops.go

```diff
@@ -4976,7 +4982,9 @@ func (fbo *folderBranchOps) registerAndWaitForUpdates() {
 					fbo.log.CDebugf(ctx, "Abandoning updates since we can't "+
 						"read the newest metadata: %+v", err)
 					fbo.status.setPermErr(err)
-					cancel()
+					// No need to lock here, since `cancelUpdates` is
+					// only set within this same goroutine.
```

Comment:
I don't understand this comment. What if a `ClearPrivateFolderMD` call happens in the meantime?